shell/platform/darwin/macos/framework/Headers/FlutterDartProject.h

```diff
@@ -9,6 +9,8 @@
 
 #import "FlutterMacros.h"
 
+NS_ASSUME_NONNULL_BEGIN
```

Comment:
Really?  We should, though not for this PR.

Reply:
The background is that the macOS embedding started life as code I was more or less the sole maintainer of, in a separate project, and I really don't like the macros. I would much rather everything be annotated so that everything is locally obvious. It also removes the conflation created by the macro between "the author thought about this, and it's non-nullable" vs. "the author forgot to think about this, so the API claims non-nullability just by default even though it's a lie, but everything still compiles because ObjC nullability is a sign not a cop". (I saw a lot of the latter in readability reviews over the years.)

But if you and @cbracken want to mass-change it, I'll just be quietly slightly sad 🙂 I do think having the engine be consistent one way or the other overall would be good.